libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -1116,4 +1160,51 @@ void AP_UAVCAN::update_mag_state(uint8_t node, uint8_t sensor_id)
     }
 }
 
+AP_UAVCAN::EscStatus_Data* AP_UAVCAN::escstatus_getptrto_data(uint8_t id)
+{
+    // I think the esc_index are continues, by how ArduPilot works
```

Comment:
"I, me, he, her" please rephrase.
Also please find out if we can jump or not, so on...